leafsmart-front-end/src/components/buttons/AddFavourite.js

```diff
@@ -1,8 +1,28 @@
+import { useAuth } from "../../contexts/AuthContext";
+import { useEffect, useState } from "react";
+
 const AddFavourite = (props) => {
+  const [geoID, setGeoID] = useState(null)
+  const [displayName, setDisplayName] = useState("")
+  const [cityName, setCityName] = useState("")
+
+  const { saveFavourites } = useAuth();
+
+  useEffect(() => {
+    setGeoID(props.geonameId);
+    setDisplayName(props.cName);
+    setCityName(props.cName);
```

Comment:
Can we destructure out these props at the top of the component? Will make understanding what props are expected to be passed in much easier! 
eg. `const { geonameId, cName } = props;`